app/src/main/kotlin/com/github/opensmsforwarder/processing/forwarder/SmsForwarderService.kt

```diff
@@ -108,10 +112,12 @@ class SmsForwarderService : Service() {
                             recipientsRepository.insertOrUpdateRecipient(
                                 recipient.copy(isForwardSuccessful = true)
```

Comment:
isForwardSuccessful - is this filed still nedded here?

Reply:
Yes, when we open the application again, we see an error in the list of recipients

Reply:
isForwardSuccessful - after refactoring it will be dropped